### ISSUE TITLE
Markdown preview styles example

### ISSUE THREAD

**Issue by vr8hub** [Microsoft/vscode#47171]
- VSCode Version: 1.21.1
- OS Version: OSX 10.12.5

There does not appear to be any documentation on the CSS used for Markdown Preview. The [VSCode Markdown page](https://code.visualstudio.com/docs/languages/markdown) has one paragraph about substituting your own CSS, but has no information on the CSS itself. We can't write our own CSS if we don't know what style elements (classes, ID's, etc.) are used in the preview window. 

Even looking at one of the replacements, i.e. the GitHub style extension, there are lots of classes used in there that are undocumented (at least I can't find any documentation on them.)

Documentation in the form of a sample MDPreview.CSS with comments that was installed by default would be greatly appreciated.

**Comment by mjbvz** [comment]
You can pretty much write normal css:

![screen shot 2018-04-04 at 1 37 04 pm](https://user-images.githubusercontent.com/12821956/38333378-4a8116bc-380d-11e8-89ec-a70d8320013f.png)

Use the `Developer: Toggle webview developer tools` to see the html itself. It's just markdown-it output.

You likely don't need to know any of the special classes like `.vscode-body` or `.code-active-line` for most common uses case. These are mainly useful for markdown extension authoring and even then you probably don't need them. 

Are there specific problems  you are running into with styling?

**Comment by mvolkmann** [comment]
I tried this. In the Elements tab of the devtools window I navigated to
#workbench.main.container
#workbench.parts.editor
.multiple-groups
.editor-two
.container
.editor-container
That contains <div aria-flowto="webview-0"></div>.
The trail ends there. It doesn't show what is inside that webview and that's what we need to style.
How can we see the elements and CSS classes used inside that?

**Comment by vr8hub** [comment]
@mjbvz, no I haven't run into problems because I didn't start because I didn't know what elements to use for styling. :)
See @mvolkmann's message, which I just confirmed. There's nothing visible but a big container for the whole preview window. How would we know what to style within that container, when the container itself isn't showing anything? (That's actually worse than I thought; I was concerned about having to go spelunking into the developer tools for each individual element on the page, but it turns out we can't even see the individual elements.)

**Comment by usernamehw** [comment]
https://github.com/electron/electron/issues/2285#issuecomment-123066883

**Comment by vr8hub** [comment]
Sorry, I don't know what I'm supposed to do with that. I'm using an editor, not trying to develop one. I know how to right-click and Inspect element (which we can't do anymore; now we have to use Help/Toggle Developer Tools, which is fine) and find what I'm looking for. I'm not an Electron developer, and I don't want to be one.

And we shouldn't have to be. Every other MD out there (that I'm familiar with anyway, which is around a dozen) documents what their preview elements are. Sometimes that doc is as simple as a sample CSS file with comments. All we're asking is that VS Code do the same.

**Comment by usernamehw** [comment]
Wait, what? There is already a command for that. https://github.com/Microsoft/vscode/issues/47171#issuecomment-378737813.

**Comment by mvolkmann** [comment]
Hey, that works!
Here are some things I learned.
You can set the background color to white with
```css
body {
  background-color: white;
}
```
You can set header colors with
```css
h1 { /* same for h2, h3, ... */
  color: red;
}
```
You can style code fences like back-tick back-tick back-tick js with these:
```css
.hljs > code > div { /* not trivial to discover */
  background-color: linen;
  color: black;
}
.hljs-comment { /* and more like this */
  color: green;
}
```

**Comment by vr8hub** [comment]
You've been using this too long, you're making a lot of assumptions about what I know. As the song says, I don't know much. :)

I'm not running the wrong command, I'm not running any command. (I haven't been using VS Code that long, but for however long it's been, I haven't used the command palette for anything. I haven't needed to.)
I did exactly what I said I did — chose Help/Toggle Developer Tools from the menu. 
But I intuited from your "running" that you meant from the command palette. Opening it and search for open did indeed find what you're talking about, and it does indeed show what we're looking for.

So thank you very much.
